code/game/objects/items/devices/radio/intercom.dm

```diff
@@ -19,6 +19,20 @@
 	STOP_PROCESSING(SSobj, src)
 	return ..()
 
+/obj/item/device/radio/intercom/attackby(obj/item/I, mob/living/user, params)
+	if(istype(I, /obj/item/weapon/wrench))

```

Comment:
Oh boy! I'm just going to run around wrenching all intercoms off the walls!
It should atleast take like a welder or something.